### PR TITLE
Modified index.css for desktop version

### PR DESCRIPTION
**Pull request by Balikuddembe**
# Hi 👋 
In this milestone_4,i have used media queries in my CSS file to ensure that my website displays correctly on a desktop version.I considered that my design matches the Figma template 1 of my choice .The break point for this version is 768px or more.I have used Grid and flex to layout the elements on the page.

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -160,13 +160,13 @@
...
     align-items: center;
+    margin-top: 9px;
   }
 
   .card-2 {
     display: flex;
     gap: 46px;
     align-items: center;
-    justify-content: flex-end;
   }
```
Comment (mukhammadrizooff) [inline index.css]: - [ ] [REQUIRED] Please add the `hover` effect(You need to add a border with the correct color and when hovering the card `border` should be lighting) to all cards like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848).

image

File: index.css [hunk 2]
```diff
@@ -160,13 +160,13 @@
     display: flex;
     gap: 46px;
     align-items: center;
+    margin-top: 9px;
   }
 
   .card-2 {
     display: flex;
     gap: 46px;
     align-items: center;
```
Comment (mukhammadrizooff) [inline index.css:168]: - [ ] [REQUIRED] Your `card` elements are positioned correctly but they have some extra space errors. Please correct the errors `highlighted` in the image below:

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @Balikuddembe, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Please choose correct color `#172B4D`✔️ for `textarea`.

image

_Check the comments under the review._

## Optional suggestions
- [ ] [OPTIONAL] Please implement several commits! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 1 above)
Comment (Balikuddembe) [reply index.css]: # Hi @mukhammadrizooff 👋 
This hover effect part on the cards, I  am supposed to do it with a coding partner as indicated in the activity of animations and transitions and it is clearly stated that i won't be reviewed by the code reviewer. Refer https://github.com/microverseinc/curriculum-html-css/blob/main/animations_exercise.md

**Review by mwanawabangona — APPROVED** [review]
Hi @Balikuddembe,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes !! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**